src/simulator/simulator.cpp

```diff
@@ -119,11 +119,27 @@ void Simulator::init(ros::NodeHandle& n) {
       "/simulator_visualization", 6);
   posMarkerPublisher = n.advertise<visualization_msgs::Marker>(
       "/simulator_visualization", 6);
+  objectLinesPublisher = n.advertise<visualization_msgs::Marker>(
+      "/simulator_visualization", 6);
   truePosePublisher = n.advertise<geometry_msgs::PoseStamped>(
       "/simulator_true_pose", 1);
   localizationPublisher = n.advertise<geometry_msgs::Pose2D>(
       "/localization", 1);
   br = new tf::TransformBroadcaster();
+
+  this->loadObject();
+}
+
+void Simulator::loadObject() {
```

Comment:
It comes to mind that we probably need a more general way to do this eventually. This is fine for now, but should be included as an issue.

Reply:
I'll add an TODO line there and create the issue after this branch is merged